### ISSUE TITLE
TransactionManagementError when writing database is not "default"

### ISSUE THREAD

**Issue by hmarty-ea** [jazzband/django-axes#1084]
When starting atomic transactions in `AxesDatabaseHandler`, the database is not specified. This can cause a `TransactionManagementError` if the writing database is not the default one after `select_for_update()` is called.

The solution would be to call `db.router.db_for_write()` to get the database name:

```
with transaction.atomic(using=db.router.db_for_write(AccessAttempt)):
    (
        attempt,
        created,
    ) = AccessAttempt.objects.select_for_update().get_or_create(
...
```

```[tasklist]
### Tasks
```

**Comment by aleksihakli** [comment]
Good catch @hmarty-ea! Would you happen to have the opportunity for opening a PR for fixing this?

**Comment by aleksihakli** [comment]
@hmarty-ea this has been fixed in 6.1.1 by @hirotasoshu, thank you :)
